apstools/beamtime/apsbss_ioc.sh

```diff
@@ -127,7 +132,10 @@ function start() {
         cd "${IOC_STARTUP_DIR}"
         # Run SESSION_NAME inside a screen session
         CMD="screen -dm -S ${SESSION_NAME} -h 5000 ${START_IOC_COMMAND}"
-        $CMD
+        ${CMD}
+        sleep 1
+        ${CAPUT} ${IOC_PREFIX}ioc_host ${HOSTNAME}
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/BCDA-APS/apstools/pullRequest?prid=5863829)